tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -48,7 +57,66 @@
     fake_beard_api_request,
     fake_magpie_api_request,
 )
-from utils import get_halted_workflow
+from utils import (
+    _delete_record,
+    get_halted_workflow,
+)
+
+
+@pytest.fixture(scope='function')
+def workflow(workflow_app):
+    workflow_object = workflow_object_class.create(
+        data={},
+        id_user=1,
+        data_type="hep"
+    )
+    workflow_object.save()
+    db.session.commit()
+    workflow_object.continue_workflow = lambda **args: True
+
+    yield workflow_object
+
+    WorkflowObjectModel.query.filter_by(id=workflow_object.id).delete()
+    db.session.commit()
+
+
+@pytest.fixture(scope='function')
+def insert_journals_in_db(workflow_app):
+    """Temporarily add few journals in the DB"""
+    from inspirehep.modules.migrator.tasks import record_insert_or_replace  # imported here because it is a Celery task
```

Comment:
you don't import this here. It's the other way around: if you need to import stuff inside a celery task, the import should be done inside the task, as it will be run somewhere else.

Reply:
Moving the import outside of the test makes it stuck. @ammirate has the same problem.  @jacquerie @michamos are there some updates regarding this mistery?